src/develop/payment.md

```diff
@@ -0,0 +1,334 @@
+
+Once you have the necessary credentials, you're ready to start making GraphQL queries to the Everscale network in your TypeScript project. To cater to various development preferences, we'll provide two versions for each example: one using the Everscale SDK (follow [this guide](https://docs.everos.dev/ever-sdk/guides/installation/add_sdk_to_your_app) for setup instructions) and another using [Axios](https://axios-http.com/), a popular HTTP client for JavaScript.
+
+## Setting up code generation
+
+First, add both the `graphql` and `@graphql-codegen/cli` packages to your project's dependencies:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="npm" label="npm" default>
+
+```shell
+npm install graphql
+npm install -D typescript
+npm install -D @graphql-codegen/cli
+npm install -D @graphql-codegen/typescript
+```
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+
+```shell
+yarn add graphql
+yarn add -D typescript
+yarn add -D @graphql-codegen/cli
+yarn add -D @graphql-codegen/typescript
+```
+  </TabItem>
+</Tabs>
+
+Next, create a file named `codegen.ts` in your project's root folder containing the following (adjust as needed):
+
+```typescript
+import type {CodegenConfig} from '@graphql-codegen/cli'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
+
+const schemaUrl = `https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: [
+    {
+      [schemaUrl]: {
+        headers: {
+          Authorization: `Basic ${PROJECT_SECRET_BASE64}`,
+        },
+      },
+    },
+  ],
+  documents: ['src/**/*.ts'],
+  ignoreNoDocuments: true, // for better experience with the watcher
+  generates: {
+    'src/generated/graphql.ts': {
+      plugins: ['typescript'],
+    },
+  },
+  watch: true,
+}
+
+export default config
+```
+
+Finally, follow [these instructions](https://the-guild.dev/graphql/codegen/docs/getting-started/development-workflow) to add GraphQL code generation to your development workflow.
+
+## API initialization
+
+To start making queries to the GraphQL API, you'll need to configure your client. Following is an example on how you can do that using either the Everscale SDK or Axios.
+
+<Tabs>
+  <TabItem value="sdk" label="Everscale SDK" default>
+
+```typescript
+import {TonClient} from '@eversdk/core'
+import {libNode} from '@eversdk/lib-node'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
+
+TonClient.useBinaryLibrary(libNode)
+
+const client = new TonClient({
+  network: {
+    endpoints: [`https://mainnet.evercloud.dev/${PROJECT_ID}/graphql`],
+    access_key: PROJECT_SECRET_BASE64,
+  },
+})
+```
+  </TabItem>
+  <TabItem value="axios" label="Axios">
+
+```typescript
+import axios from 'axios'
+
+// configure your credentials here
+const PROJECT_ID = ''
+const PROJECT_SECRET_BASE64 = ''
```

Comment:
My experiments have shown that there must be a project secret key in HEX from https://dashboard.evercloud.dev/ (Security). And in other examples, too, otherwise I would get an "unauthorized" error.